.travis.yml

```diff
@@ -16,9 +16,7 @@ cache:
     - node_modules
 
 node_js:
-  - '8'
   - '10'
-  - '12'
 
```

Comment:
Why remove?

Reply:
You're right. We would need it soon. Reverted.